python-flask/config.py

```diff
@@ -0,0 +1,14 @@
+import os
+
+# configuration
+USERNAME = os.getenv('POSTGRES_USER', 'user')
+PASSWORD = os.getenv('POSTGRES_PASSWORD', 'pass')
+HOST = os.getenv('POSTGRES_HOST', '192.168.8.175')
+PORT = os.getenv('POSTGRES_PORT', '5432')
+DATABASE = os.getenv('POSTGRES_DB')
+SECRET_KEY = os.getenv('SECRET_KEY')
+
+SQLALCHEMY_DATABASE_URI = f"postgresql://{USERNAME}:{PASSWORD}@{HOST}:{PORT}/{DATABASE}"
```

Comment:
You could avoid all that env variables. I think it is better just pass the url as and env variable instead and load it here in this SQÆAÆCHEMY_DATABASE_URI

Reply:
added `SQLALCHEMY_DATABASE_URI` in `.env`. Using the uri in `config.py`and the `docker-compose` instead.